### ISSUE TITLE
Feature request: do the analysis n times, and show median values in the report

### ISSUE THREAD

**Issue by jakub-g** [mbalabash/estimo#23]
I just played with `estimo`, it's great and exactly something that the kind of tool I had in mind for a while. Kudos 👍 

One usual issue I've noticed that on my laptop, each time I run Chrome to do perf analysis, I get substantially different results each time. For example I run the same command 3 times and I got following results

```
    scriptParseCompile: 263.56,
    scriptEvaluation: 76.04,

    scriptParseCompile: 299.46,
    scriptEvaluation: 102.38,

    scriptParseCompile: 270.82,
    scriptEvaluation: 70.26,
```

My use case is: I'm doing some optimization, and I try to compare eval times before/after.
(Usage: `estimo -r script.js --net --latency 150 --download 1.6 --cpu --cpuRate 4`

As can be seen above, the discrepancies between each run are significant.

It would be a significant improvement to be able to run the analysis `n` times (say,`n=9`) and show the median values, and maybe also the standard deviation.

`estimo -r script.js --runs 9`

It could be also interesting to make n>1 the default. For example https://www.webpagetest.org/ uses `n=3` by default.

**Comment by mbalabash** [comment]
Hello @jakub-g 
Sorry for the long response.

Yes, results might be different each time because a lot of things happening at the same time on your device and it affects available resources.

I like the idea to run analysis N times natively and show median value. 
I'll add new option: `runs ` to CLI/JS API as you suggested 👍 
Planning to do it this week.

**Comment by mbalabash** [comment]
> and maybe also the standard deviation

I'm not sure about this. It depends on device resources at a particular moment and may change dramatically.

> It could be also interesting to make n>1 the default 

It's a good idea. But honestly, I want to save default behavior as simple as possible. And extend functionality through options.

**Comment by mbalabash** [comment]
@jakub-g I finished the first implementation of this feature ([branch](https://github.com/mbalabash/estimo/tree/2.1.0)). It works well. But it still requires some tweaks and tests. I'll spend some time on it in the next days.

**Comment by mbalabash** [comment]
@jakub-g Released in `estimo@2.1.0`
